csharp/ql/test/library-tests/csharp8/DefaultInterfaceMethods.ql

```diff
@@ -0,0 +1,10 @@
+import csharp
+
+class DefaultInterfaceMethod extends Method {
```

Comment:
Could you add a test case for this as well?

Reply:
I can't yet, because it won't compile with our current Roslyn. Perhaps I should just remove this test?

Reply:
OK to leave in for me, this will be the natural place to add the test in the future anyway.